src/data/mutation/create-user.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from "@apollo/client";
+
+export const CreateUser = gql`
```

Comment:
o sufixo é importante nesse caso, já que não daria pra saber o que o objeto `CreateUser` é
```suggestion
export const CreateUserMutation = gql`
```

Reply:
Beleza, não esqueço mais